File: _scratch/run/reverse_string/sol.js
/**
 * @param {string} s
 * @return {string}
 */
var reverseString = function(s) {
    return s.split('').reverse().join('');
};
const _l=require('fs').readFileSync(0,'utf8').split('\n');const _T=+_l[0];const _o=[];
for(let _i=0;_i<_T;_i++){const _s=(_l[1+_i]||'').replace(/\r$/,'');_o.push(String(reverseString(_s)));}
console.log(_o.join('\n'));

File: _scratch/run/reverse_string/sol.py
class Solution:
    def reverseString(self, s):
        return s[::-1]
import sys
_l=sys.stdin.read().split('\n')
_T=int(_l[0]);_o=[]
for _i in range(_T):
    _s=_l[1+_i] if 1+_i<len(_l) else ""
    _o.append(str(Solution().reverseString(_s)))
print('\n'.join(_o))